src/toolkits/activity_classification/activity_classifier.hpp

```diff
@@ -32,7 +32,7 @@ class EXPORT activity_classifier: public ml_model_base {
 
   void train(gl_sframe data, const std::string& target_column_name,
              const std::string& session_id_column_name,
-             variant_type validation_data,
+             variant_type validation_data, bool has_data_augmentation,
```

Comment:
Could we move the parameter into the options dictionary? Let's also use "use_data_augmentation", since we're giving an instruction, not describing the current state of something